### PR TITLE
add a link to www.dartlang.org

### LINKED ISSUE #412: add a link to dartlang.org

### PR DESCRIPTION
**Pull request by devoncarew**
Add a link to www.dartlang.org; fix #412.

@kwalrath, @Sfshaza

### CODE REVIEW

**Comment by devoncarew** [comment]
screen shot 2015-05-12 at 9 13 14 pm

(TBR)

File: web/index.html [hunk 1]
```diff
@@ -118,17 +118,17 @@
 
     <footer layout horizontal>
       <div>
-        <a href="https://api.dartlang.org/" target="docs">API documentation</a>
+        <a href="https://www.dartlang.org/" class="footer-item"
+            target="dartlang">www.dartlang.org</a>
+        <a href="https://api.dartlang.org/" target="dartlang">API documentation</a>
```
Comment (kwalrath) [inline web/index.html]: No class="footer-item"?

File: web/index.html [hunk 2]
```diff
@@ -118,17 +118,17 @@
...
       <div id="keyboard-button" class="keyboard footer-item"></div>
       <div>
         <a href="https://github.com/dart-lang/dart-pad/wiki/Privacy-Policy"
-            target="privacy" class="footer-item">Privacy policy</a>
+            target="repo" class="footer-item">Privacy policy</a>
         <a href="https://github.com/dart-lang/dart-pad/issues"
-            target="send_feedback">
-          Send feedback
-        </a>
+            target="repo">Send feedback</a>
```
Comment (kwalrath) [inline web/index.html]: Ah, this doesn't have class="footer-item", so I guess that's necessary only for the first one.

**Comment by kwalrath** [comment]
lgtm

File: web/index.html (hunk 1 above)
Comment (devoncarew) [inline web/index.html]: `footer-item` adds a right margin to a footer element :)

**Pull request merged**